lib/active_fedora/orders/list_node.rb

```diff
@@ -0,0 +1,134 @@
+module ActiveFedora::Orders
+  class ListNode
+    attr_reader :rdf_subject
+    attr_accessor :prev, :next, :target
+    attr_writer :next_uri, :prev_uri
+    attr_accessor :proxy_in, :proxy_for
+    def initialize(node_cache, rdf_subject, graph=RDF::Graph.new)
+      @rdf_subject = rdf_subject
+      @graph = graph
+      @node_cache = node_cache
+      Builder.new(rdf_subject, graph).populate(self)
+    end
+
+    # Returns the next proxy or a tail sentinel.
+    # @return [ActiveFedora::Orders::ListNode]
+    def next
+      @next ||=
+        begin
```

Comment:
If I remove them it behaves the same in this case, since if only has one return value. Would you prefer
```ruby
@next ||=
  if next_uri
  end
``` ?